atc/creds/vault/manager.go

```diff
@@ -90,50 +86,27 @@ func (manager *VaultManager) MarshalJSON() ([]byte, error) {
 	})
 }
 
-func toString(s interface{}) string {
-	if s == nil {
-		return ""
-	}
-	return s.(string)
-}
+func (manager *VaultManager) Config(config map[string]interface{}) error {
+	// apply defaults
+	manager.PathPrefix = "/concourse"
```

Comment:
This is a good catch, as it wasn't there in the initial implementation and I would get errors when specifying a `var_source` without `path_prefix`.